docs/cookbook/getSortedLocalDateTimes.mjs

```diff
@@ -0,0 +1,24 @@
+import assert from "assert";
+/**
+ * getSortedLocalDateTimes will sort an array of zoneless Temporal.DateTime instances by the
+ * corresponding local date and time of day (e.g., for building a conference schedule).
+ *
+ *
+ * @param {Temporal.DateTime[]} dateTimes This is a Date instance
+ * @param {boolean} direction ascending or descending order
+ * @returns {Temporal.DateTime[]} Temporal.Absolute instance
+ */
+function getSortedLocalDateTimes(dateTimes, reverse = false) {
+    let newDateTimes = dateTimes.sort(Temporal.DateTime.compare);
+
+    return reverse ? newDateTimes.reverse() : newDateTimes;
+}
+
+// Sorting some conferences without timezones
+let a = Temporal.DateTime.from({ year: 2020, day: 15, month: 2, hour: 9 }); // FontFest
```

Comment:
I'd say that sorting the start times of different conferences is not as good of an example use case as sorting the times of sessions within one conference; for one thing, the conferences are actually in different time zones!